packages/atomic/src/index.html

```diff
@@ -7,17 +7,27 @@
 
     <script type="module" src="/build/atomic.esm.js"></script>
     <script nomodule src="/build/atomic.js"></script>
-    <link rel="stylesheet" href="/build/atomic.css" />
-    <!-- Initialization example
-  <script>
-    (async () => {
-      await customElements.whenDefined('atomic-search-interface');
-      document.querySelector('#search').initialize({
-        accessToken: 'my_token',
-        organizationId: 'my_org',
-      });
-    })();
-  </script> -->
+    <script>
+      (async () => {
+        await customElements.whenDefined('atomic-search-interface');
+        const searchInterface = document.querySelector('#search');
+
+        // Initialization
+        /* searchInterface.initialize({
+          accessToken: 'my_token',
+          organizationId: 'my_org',
+        }); */
+
+        searchInterface.afterInitialization(() => {
```

Comment:
What is the reason that this needs to be configured after initialization ? 

And what would be needed for this constraint to go away ?

Reply:
My guess is there would be a hook/event of some sort, no way everything will be initialized and available right after `await customElements.whenDefined('atomic-search-interface');` we have to give time for the engine/i18n to initialize.

I'll rework the init mechanism a bit, more on that later...